CHANGES/README.rst

```diff
@@ -34,21 +31,22 @@ for the users to understand what it means.
 combined with others, it will be a part of the "news digest"
 telling the readers **what changed** in a specific version of
 the library *since the previous version*. You should also use
-reStructuredText syntax for highlighting code (inline or block),
-linking parts of the docs or external sites.
-If you wish to sign your change, feel free to add ``-- by
+*reStructuredText* syntax for highlighting code (inline or block),
+linking parts of the docs or external sites. However, you do not need to
+reference the issue or PR numbers here as *towncrier* will automatically
+add a reference to all of the affected issues when rendering the news
+file. If you wish to sign your change, feel free to add ``-- by
```

Comment:
Also, no need to reflow existing lines with sentences starting at the beginning. It makes diffs cleaner and RST knows how to merge them into a paragraph anyway.
```suggestion
linking parts of the docs or external sites.
However, you do not need to
reference the issue or PR numbers here as *towncrier* will automatically
add a reference to all of the affected issues when rendering the news
file.
If you wish to sign your change, feel free to add ``-- by
```